### ISSUE TITLE
Pop-up mismatch after an incomplete Manual Backup

### ISSUE THREAD

**Issue by ConEstArthur** [Medvedev91/timeto.me#62]
Here's how you can reproduce the same.

I'm on using timeto on android platform.
- In backup section tap on **Create**
- After it opens your default file manager, do not do anything. Just press back.
- When it switches back to the app.
- This pop-up appears

![Screenshot_20240709-163012](https://github.com/Medvedev91/timeto.me/assets/76774182/ee99e0fa-fc5a-49da-b368-d48994b4fb49)

which is fine if I would have selected **Restore** option.

---
Also If I tap on Auto Backup option, it opens up the file manager which I presume would let me select default auto backup location. But in default Files app I cannot see **Use this folder** option to actually select the location.

**Comment by Medvedev91** [comment]
> This pop-up appears

Making a backup is a very important thing. So I prefer to show an error in any unclear situation. Including if the user started creating a backup but did not complete it.

> I presume would let me select default auto backup location

Like this #61 ?

**Comment by ConEstArthur** [comment]
> Including if the user started creating a backup but did not complete it.
Yes I get that. I was just showing that the phrasing could have been better.

> Like this #61 ?

Yes, that's a feature request but I did not get what does it want me to do when I tap on Auto Backup.
It opens up file manager app then what ? ( In its current stage )

The backup section is incomplete at this stage so few annoyances are justified.

**Comment by Medvedev91** [comment]
> Yes, that's a feature request but I did not get what does it want me to do when I tap on Auto Backup.

I made this feature just to make users know about the auto-backup.

I understand that some people need to change the folder to cloud sync for example. Of course, someday it will be implemented.

**Comment by Medvedev91** [comment]
> could possibly deter even long time users

The long post about that https://github.com/Medvedev91/timeto.me/issues/160#issuecomment-3810958432 . In short - I believe that the latest implementation covers all the cases from the activities system.

> What I mean to say is we need to focus on the elements of the app which makes it an all in one app - calender, notes, checklists and ease of use ofcourse.

I believe the next release will make calendar much stronger!

**Comment by ConEstArthur** [comment]
> hmm... By the way, it is a great idea to set default timer hint as goal's time. Can you please create an issue?

Done.

**Comment by Medvedev91** [comment]
@ConEstArthur Do you mean recursive checklist like this?

https://github.com/user-attachments/assets/16f82304-00e3-41ca-8f6f-3f75a75d3194

**Comment by Medvedev91** [comment]
> recursive checklists.

Recursive, I mean when checklist triggers itself.

Okay. What we are going to do with that? :)

**Comment by Medvedev91** [comment]
"recursive checklist" is not a feature it is a side effect :)
